### PR TITLE
[Style] Update classification page layouts

### LINKED ISSUE #11427: ✨ Admin / Classifications / mostly Styling update
## ✨ Feature

Update styling of the Community pages, standardize the way things work, and bring it in line with the overall platform's look and feel.

### Out of scope for this issue (don't build this even if it's in the design)
- [ ] Moving to the new nav
- [ ] Updates to the table component
- [ ] Adding "searchable" to the classification model (and associated UI)
- [ ] Advanced (or advanced tools) tab
- [ ] Changes to table search and filter

## 🎨 Design

[Figma file](https://www.figma.com/design/Ov3ZwbSJXh7Ckqk6cefPDo/Admin---Classifications?node-id=5437-34104&t=NdEom57TwzcRXijs-1) by @Jerryescandon 

## ✅ Acceptance Criteria

- [ ] Classifications page (/admin/settings/classifications)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Remove "edit" column, add edit link to the "Name" column instead
- [ ] Create classification (/admin/settings/classifications/create)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
- [ ] Edit a community page (/admin/classifications/:id:/edit)
  - [ ] (should use same component as Create Classification page)
- [ ] New page - Classifications information (/admin/classifications/:id:)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Only one tab for now
  - [ ] Inline form for displaying information and editing

- [ ] Mobile view for all pages

- [ ] Design Review by @Jerryescandon 

## Blockers
```[tasklist]
### Blocked by
- [ ] #11344 
```

### PR DESCRIPTION
**Pull request by esizer**
🤖 Resolves #11427 

## 👋 Introduction

Updates the classification pages to use the new layout/styles.

## 🧪 Testing

1. Build the app `pnpm run dev:fresh`
2. Login as admin `admin@test.com`
3. Navigate to the classifications section `/admin/settings/classifications`
4. Check all pages and confirm they function as expected and match the designs

### CODE REVIEW

File: apps/web/src/pages/Classifications/components/ClassificationTable.tsx
[the remark this answers is not in the document]
Comment (vd1992) [reply apps/web/src/pages/Classifications/components/ClassificationTable.tsx:1]: As well as naming changes

Current

image

Figma

image

File: apps/web/src/pages/Classifications/ViewClassificationPage.tsx [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+      {
+        label: intl.formatMessage(pageTitles.classifications),
+        url: paths.classificationTable(),
+      },
+      {
+        label: pageTitle,
+        url: paths.classificationView(classification.id),
+      },
+    ],
+  });
```
Comment (vd1992) [inline apps/web/src/pages/Classifications/ViewClassificationPage.tsx:72]: Crumbs vary

Current

image

Figma (ignoring the admin dashboard one which is the same as the home link)

image

**Review by vd1992 — COMMENTED** [review]
I probably should've just let @Jerryescandon do the design review first instead, mostly just have questions below 😅

**Comment by petertgiles** [comment]
Please check my review for #11926.  I'm guessing a lot of the same things will come up here.  :smile:

File: apps/web/src/pages/Classifications/components/ClassificationTable.tsx
Comment (petertgiles) [inline apps/web/src/pages/Classifications/components/ClassificationTable.tsx:1]: The button should be labelled "Create new classification"

Comment (petertgiles) [inline apps/web/src/pages/Classifications/components/ClassificationTable.tsx:1]: Hero should be centered in mobile width

File: apps/web/src/pages/Classifications/CreateClassificationPage.tsx
Comment (petertgiles) [inline apps/web/src/pages/Classifications/CreateClassificationPage.tsx:1]: Label should be "Level number"

File: apps/web/src/pages/Classifications/UpdateClassificationPage.tsx
Comment (petertgiles) [inline apps/web/src/pages/Classifications/UpdateClassificationPage.tsx:1]: Label should be "Level number"

Comment (petertgiles) [inline apps/web/src/pages/Classifications/UpdateClassificationPage.tsx:1]: Icon should be the info cord, not cloud

**Review by petertgiles — CHANGES_REQUESTED** [review]
Cool!  A few more things to tidy up.

File: apps/web/src/pages/Classifications/UpdateClassificationPage.tsx
Comment (esizer) [reply apps/web/src/pages/Classifications/UpdateClassificationPage.tsx:1]: It is actually using a cloud in the design.

File: apps/web/src/pages/Classifications/components/ClassificationTable.tsx
Comment (esizer) [reply apps/web/src/pages/Classifications/components/ClassificationTable.tsx:1]: I'm just using the default hero and it looks centered on my end :thinking:

File: apps/web/src/pages/Classifications/UpdateClassificationPage.tsx
Comment (petertgiles) [reply apps/web/src/pages/Classifications/UpdateClassificationPage.tsx:1]: @Jerryescandon please confirm?

File: apps/web/src/pages/Classifications/components/ClassificationTable.tsx
Comment (esizer) [reply apps/web/src/pages/Classifications/components/ClassificationTable.tsx:1]: Weird, I though "new" was implied when creating something :laughing: 

Fixed in [update add label](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11901/commits/c6550f05aad00a20fa90b19819ae833b59584d65)

File: apps/web/src/pages/Classifications/CreateClassificationPage.tsx
Comment (esizer) [reply apps/web/src/pages/Classifications/CreateClassificationPage.tsx:1]: Updated all instances in [update level label to include number](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11901/commits/b9a44ce4d630db0789561060bd9f8c92a28683dd)

File: apps/web/src/pages/Classifications/components/ClassificationTable.tsx
[the remark this answers is not in the document]
Comment (Jerryescandon) [reply apps/web/src/pages/Classifications/components/ClassificationTable.tsx:1]: I would say yes, and get them over with now. But I dont know how big they are 😬

**Review by petertgiles — APPROVED** [review]

**Pull request merged**